mqtt-protocol/build.gradle.kts

```diff
@@ -6,10 +8,167 @@ kotlin {
                 implementation(project(":core"))
             }
         }
+        val multithreadMain by creating{
```

Comment:
What is the `multithreadedMain` source set?

Reply:
it's a sourceSet common for Native and JVM to avoid write the same code twice, the name can be changed for a more appropriate one if you have ideas

Reply:
Oh, I see, so I propose `jvmNativeMain` how does it sound?

Reply:
sounds good to me, i will change it immediatly